src/main/java/Objects/Packets/SYN.java

```diff
@@ -1,10 +1,20 @@
 package Objects.Packets;
 
+/**
+ * Specialized packet type for causing
+ * collisions.
+ */
 public class SYN extends Packet {
 
-    public SYN() {
-        /*setCost(1);
-        setStealth(1);*/
+    /**
+     * Constructor for SYN packet type.
+     * @param team Team that owns the packet.
+     */
+    public SYN(final String team) {
+        setPacketType("SYN");
+        setTeam(team);
+        setCost(1);
+        setStealth(1);
     }
 
```

Comment:
this code breaks the program, particularity the fileHandler when it creates the objects. That should be amended even if it is just passing null variables

Reply:
program is still broken. Culprit appears to be your setPacket function not being a setPackets function that takes in a List